[Developer setup] zotero-plugin fork npm is not public?

I'm trying to follow the developer setup to do some debugging on https://github.com/dvanoni/notero/issues/149, but I'm getting: 

```
❯ npm ci
npm WARN prepare removing existing node_modules/ before installation
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@dvanoni/zotero-plugin/1.6.0/957c40a9cfc6c368a3b8f558f6c47e232c57fb56
```

Using `nvm --lts`:

```
Now using node v14.17.6 (npm v6.14.15)
```

--- a/README.md
+++ b/README.md
@@ -358,7 +358,8 @@ The steps below should allow you to build and run Notero yourself.
     where to install the extension when running a development build.
 
 4.  In order to install the forked `zotero-plugin` package, you must
-    [log in to the GitHub Packages npm registry][github-npm-auth]:
+    [log in to the GitHub Packages npm registry][github-npm-auth] using a GitHub
+    token with the `read:packages` scope (generate one [here][read-packages-token]):
 
         npm login --scope=@dvanoni --registry=https://npm.pkg.github.com
 
@@ -387,6 +388,7 @@ The steps below should allow you to build and run Notero yourself.
 
 [generator-zotero-plugin]: https://github.com/retorquere/generator-zotero-plugin
 [github-npm-auth]: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-with-a-personal-access-token
+[read-packages-token]: https://github.com/settings/tokens/new?scopes=read:packages
 [zotero-plugin]: https://github.com/dvanoni/zotero-plugin
 [zotero-start]: https://github.com/dvanoni/zotero-plugin/blob/master/bin/start.py
 

--- a/README.md
+++ b/README.md
@@ -335,8 +335,8 @@ See below for descriptions of how you can use the different views.
 
 ## Development
 
-Notero was scaffolded with [generator-zotero-plugin](https://github.com/retorquere/generator-zotero-plugin)
-and is built with [zotero-plugin](https://github.com/retorquere/zotero-plugin).
+Notero was scaffolded with [generator-zotero-plugin][generator-zotero-plugin]
+and is built with a fork of [zotero-plugin][zotero-plugin].
 
 ### Local Setup
 
@@ -353,29 +353,42 @@ The steps below should allow you to build and run Notero yourself.
     has a directory named `extensions`.
 
 3.  Create a `zotero-plugin.ini` file as described in the
-    [zotero-plugin README](https://github.com/retorquere/zotero-plugin#readme).
+    [zotero-plugin][zotero-plugin] README.
     This file is used by the [`zotero-start`][zotero-start] command to determine
     where to install the extension when running a development build.
 
-4.  Install dependencies:
+4.  In order to install the forked `zotero-plugin` package, you must
+    [log in to the GitHub Packages npm registry][github-npm-auth]:
+
+        npm login --scope=@dvanoni --registry=https://npm.pkg.github.com
+
+5.  Install dependencies:
 
         npm ci
 
-5.  Build the plugin and start Zotero with it installed:
+6.  Perform an initial build of the plugin to create the `build` directory:
+
+        npm run build
+
+7.  Start Zotero with the plugin installed:
 
         npm start
 
     The `start` script runs [`zotero-start`][zotero-start] which performs a
     number of steps:
 
-    1.  Executes `npm run build` to build the plugin into the `build/` directory.
-    2.  Writes a new `.xpi` file containing the absolute path to the `build/` directory.
+    1.  Executes `npm run build` to build the plugin into the `build` directory.
+    2.  Writes a file containing the absolute path to the `build` directory into
+        the `extensions` directory in the Zotero profile directory.
     3.  Starts Zotero with the profile specified in `zotero-plugin.ini` and the
         following command line arguments:
 
-            -purgecaches -jsconsole -ZoteroDebugText -datadir profile
+            -purgecaches -jsconsole -ZoteroDebugText -debugger -datadir profile
 
-[zotero-start]: https://github.com/retorquere/zotero-plugin/blob/master/bin/start.py
+[generator-zotero-plugin]: https://github.com/retorquere/generator-zotero-plugin
+[github-npm-auth]: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-with-a-personal-access-token
+[zotero-plugin]: https://github.com/dvanoni/zotero-plugin
+[zotero-start]: https://github.com/dvanoni/zotero-plugin/blob/master/bin/start.py
 
 ### Releasing a New Version
 
@@ -391,6 +404,5 @@ The steps below should allow you to build and run Notero yourself.
         git push --follow-tags
 
 3.  GitHub Actions will run the [`release`](.github/workflows/release.yml)
-    workflow upon any new commit. This workflow will build the `.xpi` file and
-    then use the [`zotero-plugin-release`](https://github.com/retorquere/zotero-plugin/blob/master/bin/release.ts)
-    command from `zotero-plugin` to create a GitHub release with the `.xpi` file.
+    workflow upon push of a version tag. This workflow will build the `.xpi`
+    file and then create a GitHub release with the `.xpi` file.

--- a/README.md
+++ b/README.md
@@ -355,7 +355,18 @@ The steps below should allow you to build and run Notero yourself.
 3.  Create a `zotero-plugin.ini` file as described in the
     [zotero-plugin][zotero-plugin] README.
     This file is used by the [`zotero-start`][zotero-start] command to determine
-    where to install the extension when running a development build.
+    where to install the plugin when running a development build.
+
+    **Example**
+
+    ```ini
+    [profile]
+    name = dev
+    path = /Users/dvanoni/Library/Application Support/Zotero/Profiles/sekg0jb3.dev
+
+    [plugin]
+    source = build
+    ```
 
 4.  In order to install the forked `zotero-plugin` package, you must
     [log in to the GitHub Packages npm registry][github-npm-auth] using a GitHub

